app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,9 @@ def new
   def login
     user = User.find_by(username: params[:username])
     if user && user.authenticate(params[:password])
-      continue_to_phone_verification(user)
+      session[:user_id] = user.id
+      session[:current_folder_id] = user.root
+      redirect_to root_path
```

Comment:
they should only be able to log in if they are ```confimred!``` i'll make this change and create a new pr